cfgov/v1/util/ref.py

```diff
@@ -46,6 +46,7 @@
     ('rule-under-dev', 'Rule Under Development'),
     ('story', 'Story'),
     ('ask', 'Ask CFPB'),
+    ('cfpb-researchers', 'CFPB Researchers'),
```

Comment:
Can the page types list be alphabetized?

Reply:
I wondered that too but was hesitant to reorder this list without understanding how else it's used or what content editor expectations are around this list. Who would be the best person to opine on this?

Reply:
I'd defer to @Scotchester but it seems to me alphabetical (by displayed text, the second parameter) would be the way to go, since that would make it easy to find what you're looking for in the Wagtail editor dropdown.

Reply:
I agree that that would be the best way to alphabetize it, and I can't see why it would present a problem to do so. Surely no code relies on the order of the list.